pyproject.toml

```diff
@@ -6,8 +6,12 @@ requires = [
     "setuptools>=34.4",
 
     # Plugins
-    "setuptools_scm[toml]>=3.5",
-    "setuptools_scm_git_archive>=1.1",
+    "setuptools-scm >= 3.5; python_version >= '3.7'",
```

Comment:
Why is this not 7.0+? That's the version that first integrated support for `.git_archival.txt`.